### DISCUSSION TITLE
multiple subvolume groups

### DISCUSSION THREAD

**Discussion by takyon77** [ceph/ceph-csi#3227] [General]
Hi,

I would like to use more than one subvolume group in ceph-csi. If my understanding is correct, this was added in the code by issue 1175 (https://github.com/ceph/ceph-csi/pull/1175) but I'm not sure how to use it because the values to configure ceph-csi request a single string for the subvolumeGroup apparently. Am I missing something in the configuration ? Ideally I'd like to use a subvolume group per openshift project, is this can be done somehow?

Thanks,

Chris.

**Comment by Madhu-1** [comment]
https://github.com/ceph/ceph-csi/issues/1884#issuecomment-787665156 help to create a cephcsi configmap with multiple subvolumegroups to use the specific subvolumegroup you need to choose the right clusterID and specify it in storageclass.

**Comment by takyon77** [comment]
Thanks @Madhu-1, I'll give it a try.
